main.py

```
@@ -16,8 +26,24 @@ async def main():
+                try:
+                    # This will run the agent and create a HoneyHive experiment
+                    response = await run_agent_with_experiment(prompt)
+                    logger.info("HoneyHive experiment created for this query")
+                except Exception as e:
+                    logger.error(f"Error running agent with HoneyHive experiment: {e}")
+                    # Fallback to regular agent run
+                    logger.warning("Falling back to regular agent run without HoneyHive experiment")
+                    response = await agent.run(prompt)
+            else:
+                # Regular agent run without HoneyHive
+                response = await agent.run(prompt)
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)